src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field block">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="aaa">Surname:</label>
```

Comment:
What is "aaa"? Try to provide more descriptive values)
You should also set this id for input, to connect it with label.
Or just wrap it into label, so that you don't need "for" attribute